.github/workflows/build.yaml

```diff
@@ -0,0 +1,23 @@
+name: Pipenv Coveralls Test
+
+on: ["push", "pull_request"]
```

Comment:
Is this push intentional?

Reply:
It's a public repo so I don't think the minutes eaten up by it cost Lob anything.